.github/workflows/format.yaml

```
@@ -21,11 +21,34 @@ on:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
       - uses: ./.github/actions/dump-context
+  detect-ci-container:
+    permissions:
+      contents: read
+    permissions:
```

actionlint (1.7.7)

key "permissions" is duplicated in "detect-ci-container" job. previously defined at line:40,col:5

(syntax-check)